diagnostic.md

```diff
@@ -111,12 +185,36 @@ JSON.
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut =  function(event) {
+  event.preventDefault();
+  let $itemIdImput = $(event.target).find('[name="item_id"]');
+  let itemId = $itemIdImput.val();
+  let item = new FormData(event.target);
+  $.ajax({
+    url: 'http://www.example.com/donuts' +itemId,
+    type: 'PATCH',
+    contentType: false,
+    processData: false,
+    data: item
+  })
+  .done(logResponseBody)
+  .fail(logRequestError);
+  itemIdImput.val('');
+};
+
```

Comment:
Again, this is a good solution for using `FormData`, in the next question. In this question, we wanted you to hard-code some data.